gatk/4.beta.1-3.5/Dockerfile

```diff
@@ -4,10 +4,11 @@ MAINTAINER Yuankun Zhu (zhuy@email.chop.edu)
 ENV GATK4_VERSION 4.beta.1
```

Comment:
It won't let me comment high, but here: `MAINTAINER Yuankun Zhu (zhuy@email.chop.edu)` might be good to switch to `LABEL` format while you're already here, to bring this back up to spec.Do you think doing a quick test run of `tools/gatk_haplotypecaller.cwl` would be good to ensure this works, or is that overkill?

Reply:
Updated.